decompiler/config/all-types.gc

```diff
@@ -10925,82 +10925,67 @@
-;    (poly-count      uint16         :offset-assert 14)
-;    (base-trans      vector :inline :offset-assert 16)
-;    (vertex-count    uint8          :offset-assert 28)
-;    (vertex-data-qwc uint8          :offset-assert 29)
-;    (total-qwc       uint8          :offset-assert 30)
-;    (unused          uint8          :offset-assert 31)
-;    )
-;   :method-count-assert 9
-;   :size-assert         #x20
-;   :flag-assert         #x900000020
-;   )
-
-; (deftype collide-fragment (drawable)
-;   ((id      int16          :offset-assert 4)
-;    (bsphere vector :inline :offset-assert 16)
-;    (mesh    basic          :offset-assert 8)
-;    )
-;   :method-count-assert 18
-;   :size-assert         #x20
-;   :flag-assert         #x1200000020
-;   (:methods
-;     (dummy-9 () none 9)
-;     (dummy-10 () none 10)
-;     (dummy-11 () none 11)
-;     (dummy-12 () none 12)
-;     (dummy-13 () none 13)
-;     (dummy-14 () none 14)
-;     (dummy-15 () none 15)
-;     (dummy-16 () none 16)
-;     (dummy-17 () none 17)
-;     )
-;   )
+(deftype collide-frag-vertex (vector)
+  ()
+  :method-count-assert 9
+  :size-assert         #x10
+  :flag-assert         #x900000010
+  )
 
-; (deftype drawable-inline-array-collide-fragment (drawable-inline-array)
-;   ((id      int16           :offset-assert 4)
-;    (bsphere vector  :inline :offset-assert 16)
-;    (length  int16           :offset-assert 6)
-;    (data    UNKNOWN 1       :offset-assert 36)
-;    )
-;   :method-count-assert 18
-;   :size-assert         #x44
-;   :flag-assert         #x1200000044
-;   (:methods
-;     (dummy-9 () none 9)
-;     (dummy-10 () none 10)
-;     (dummy-11 () none 11)
-;     (dummy-12 () none 12)
-;     (dummy-13 () none 13)
-;     (dummy-14 () none 14)
-;     (dummy-15 () none 15)
-;     (dummy-16 () none 16)
-;     (dummy-17 () none 17)
-;     )
-;   )
+(deftype collide-frag-mesh (basic)
+  ((packed-data     uint32         :offset-assert 4)
+   (pat-array       uint32         :offset-assert 8)
+   (strip-data-len  uint16         :offset-assert 12)
+   (poly-count      uint16         :offset-assert 14)
+   (base-trans      vector :inline :offset-assert 16)
+   (vertex-count    uint8          :offset 28)
+   (vertex-data-qwc uint8          :offset 29)
+   (total-qwc       uint8          :offset 30)
+   (unused          uint8          :offset 31)
+   )
+  :method-count-assert 9
+  :size-assert         #x20
+  :flag-assert         #x900000020
+  )
 
+(deftype collide-fragment (drawable)
+  ((mesh    basic          :offset 8)
+   )
+  :method-count-assert 18
+  :size-assert         #x20
+  :flag-assert         #x1200000020
+  (:methods
+    (dummy-9 () none 9)
+    (dummy-11 () none 11)
+    (dummy-12 () none 12)
+    (dummy-13 () none 13)
+    (dummy-14 () none 14)
+    (dummy-15 () none 15)
+    (dummy-17 () none 17)
+    )
+  )
 
-;; - Symbols
+(deftype drawable-inline-array-collide-fragment (drawable-inline-array)
+  ((data    uint8       36) ; NOTE - guess!
```

Comment:
I think I figured out the pattern for these `drawable-inline-array`s.  This would be:
```lisp
(deftype drawable-inline-array-collide-fragment (drawable-inline-array)
  ((data    collide-fragment 1 :inline      :offset-assert 32)
   (pad uint32) ;; ending in inline basic always results in 4 byte pad.
   )
...
```

This should make the size correct and make the inspect for `data` look correct.